external-crates/move/crates/move-vm-runtime/src/data_cache.rs

```diff
@@ -61,10 +61,7 @@ impl<S: MoveResolver> TransactionDataCache<S> {
 
             if !modules.is_empty() {
                 change_set
-                    .add_account_changeset(
-                        addr,
-                        AccountChangeSet::from_modules(modules),
-                    )
+                    .add_account_changeset(addr, AccountChangeSet::from_modules(modules))
```

Comment:
Artifact of `cargo fmt`, but we _should_ use it.

Reply:
As discussed, let's see if we can fix it

Reply:
Fixed as of #18738